fluid_onnx/ops.py

```diff
@@ -459,8 +421,29 @@ def randomuniformlike_op():
     pass
 
 
-def reciprocal_op():
-    pass
+def reduce_ops(op_type, operator, block):
```

Comment:
A good explainer comment on these bigger op functions (things that support more than one fluid op) would be awesome

Reply:
Good suggestion.